assets/js/search.js

```diff
@@ -3,7 +3,7 @@ const idx = [
   {
     "link": "{{ .Permalink }}",
     "title": "{{ .Title }}",
-    "body": `{{ .PlainWords }}`.toLowerCase(),
+    "body": "{{ .PlainWords }}".toLowerCase(),
```

Comment:
This change introduced a bug that I had solved earlier. The reason I used template strings is because they automatically escape `"`  from a page's content